package.json

```diff
@@ -147,10 +147,6 @@
       "path": "./api/build/static/webapp/main.js",
       "maxSize": "950 kB"
     },
-    {
-      "path": "./api/build/static/webapp/polyfills-es5.js",
-      "maxSize": "55 kB"
```

Comment:
I am a little nervous about this.  I read the [blogpost](https://blog.angular.io/angular-v13-is-now-available-cce66f7bc296) you linked to above and agree that it only describes this as necessary for supporting IE11.  However, if it was only needed for IE, I would not have expected it to be present before this upgrade since our [`.browserlistrc`](https://github.com/medic/cht-core/blob/392ce01762dc90958e38e8e61c05f57550d2f819/.browserslistrc) is only targeting Chome/FF (no IE).  

What is more, I double-checked the [Browser Support](https://www.w3schools.com/js/js_es6.asp) listed for `es6` and it specified Chrome 58 as the first version with support for `es6`.  This seems to mean that Chrome 53-57 would still require `es5`.  IDK, maybe I am totally missing something here, but at least we need to make sure we test this carefully on some devices with really old webview versions....

Reply:
> This seems to mean that Chrome 53-57 would still require es5

I think we're assuming that core v4.x.x supports [Chrome and Firefox latest](https://docs.communityhealthtoolkit.org/core/releases/#dependencies) with the intention to start using modern js features. I'd expect partners to be at least 80+

Reply:
Related Warning I see when building the webapp:

> One or more browsers which are configured in the project's Browserslist configuration will be ignored as ES5 output is not supported by the Angular CLI. Ignored browsers: chrome 60, chrome 59, chrome 58, chrome 57, chrome 56, chrome 55, chrome 54, chrome 53

Reply:
Good catch, I've updated the `.browserlistrc` and the `isUsingSupportedBrowser` functions to support:
```
chrome: '>=80', // Chrome 80 was released on February 4, 2020; for desktop and Android.
firefox: '>=98', // Firefox 93 was released on October 5, 2021; for desktop and Android.
```
Chrome v80 seems the safest after comparing the es2020 features with the browser support. Firefox stays the same as before.  That gives 3 years of Chrome versions to today.

Reply:
Angular actually [suggests](https://angular.io/guide/build#configuring-browser-compatibility) 
```
last 1 Chrome version
last 1 Firefox version
```

But it's probably too much :)

Reply:
Angular detected the new Chrome version in `.browserlistrc` and the following warning is gone.
> One or more browsers which are configured in the project's Browserslist configuration will be ignored as ES5 output is not supported by the Angular CLI. Ignored browsers: chrome 60, chrome 59, chrome 58, chrome 57, chrome 56, chrome 55, chrome 54, chrome 53


However, this other warning is still showing up:
> TypeScript compiler options "target" and "useDefineForClassFields" are set to "ES2022" and "false" respectively by the Angular CLI. To control ECMA version and features use the Browerslist configuration. For more information, see https://angular.io/guide/build#configuring-browser-compatibility

Even if we set the `.browserlistrc` as Angular does, or move that file inside webapp, the warning keeps coming up. The app still working fine so far.